ext/storage_sources/s3_store/README.md

```diff
@@ -0,0 +1,77 @@
+# WiredTiger's S3 Store Extension
+
+## Building and running 
+
+This is a guide to build WiredTiger with the S3 extension enabled.
+
+##### **First** Build method - meeting AWS dependency through system installed library
+
+This method will find a locally installed version of the AWS S3 and will not require a download stage during build.
```

Comment:
This method will find an existing system installation of the AWS S3 and will not require a download stage during build.